Formula/platformio.rb

```diff
@@ -3,8 +3,8 @@ class Platformio < Formula
 
   desc "An open-source ecosystem for embedded development"
   homepage "https://platformio.org/"
-  url "https://files.pythonhosted.org/packages/b5/b6/832b92113764feae278376dd0f39f0a3d4321c24b9ff1ff9c9777ba23b8d/platformio-4.1.0.tar.gz"
-  sha256 "4552086d82fb4d7022abeb620499314249548d084b37ddf446a7ee3099b13c21"
+  url "https://files.pythonhosted.org/packages/31/6b/6050e999bb94b7b0db86518a345fe118f8727e61f1d261df121d012878cf/platformio-4.2.0.tar.gz"
+  sha256 "bfe062fbe014ed4776f3e56491324bda9049de7a1338ea07972407e324623187"
   revision 1
```

Comment:
Please remove this line

Reply:
Done!
P.S: I didn't add it, it was added by brew bot :(